tests/dag_tests.sh

```diff
@@ -63,8 +63,8 @@ if [ $1 = "alignment" ]; then
   printf "\ndownload default\n"
   seq2science run download-fastq -n --configfile tests/$WF/default_config.yaml --snakemakeOptions quiet=True | tee tests/local_test_results/${1}_dag
   assert_rulecount $1 runs2sample 3
-  assert_rulecount $1 ena2fastq_SE 5
-  assert_rulecount $1 ena2fastq_PE 1
+  assert_rulecount $1 'ena2fastq_SE|sra2fastq_SE' 5
+  assert_rulecount $1 'ena2fastq_PE|sra2fastq_PE' 1
```

Comment:
Nice, but also not nice. ENA was down couple days, so that is annoying. However if something gets broken in ena downloading we won't realize

Reply:
Samples will always be downloaded from ENA if it is up. We'd need some samples available only from SRA if you want to properly test both sets of rules. Ill make a separate issue for this.